manifests/init.pp

```diff
@@ -54,15 +54,15 @@
 
   Integer[0] $hosts_queue_workers = 1,
 ) {
-  package { 'katello':
-    ensure => installed,
-  }
-
   class { 'katello::params':
     candlepin_oauth_key    => $candlepin_oauth_key,
     candlepin_oauth_secret => $candlepin_oauth_secret,
   }
 
+  package { $katello::params::meta_package:
```

Comment:
Can we keep this at the same line as it used to be, to make the diff a bit smaller?

Reply:
No, as then the variable is not defined yet, I tried ;)

Reply:
Oh yes, of course.